handel/src/services/sqs/params-schema.json

```diff
@@ -0,0 +1,130 @@
+{
+    "$schema": "http://json-schema.org/schema#",
+    "type": "object",
+    "properties": {
+        "type": {
+            "type": "string",
+            "description": "The service type (should always be 'sqs')",
+            "pattern": "sqs",
```

Comment:
You'll want to put the '^' and '$' characters on the two ends of this pattern so that it will only accept the string 'sqs'